WordPress/WordPressShareExtension/PostStatusPickerViewController.swift

```diff
@@ -0,0 +1,103 @@
+import Foundation
+import UIKit
+import WordPressShared
+
+class PostStatusPickerViewController : UITableViewController
+{
+    // MARK: - Initializers
+    init(statuses : [String: String]) {
+        assert(statuses.count > 0, "Let's show at least one status!")
+        
+        // Note:  We'll store the sorted Post Statuses, into an array, as a (Key, Description) tuple.
+        self.sortedStatuses = statuses.sort { $0.1 < $1.1 }
+        super.init(style: .Plain)
+    }
+    
+    required init?(coder: NSCoder) {
```

Comment:
The only reason you have to implement that initializer at all is because you want to override the existing (public) UIViewController initializer. Overriding that privately makes no sense.

If you want to prevent accidental usage, you can use `@available(*, unavailable)`. Although in that case it won't help you much as this is would be used from Storyboards, which would look at your `@available` and laugh :wink: